portfolio/src/main/webapp/animation.js

```diff
@@ -0,0 +1,25 @@
+let p;
```

Comment:
Particularly for global variables (but not e.g. for counter variables in for loops), I would suggest a longer name, e.g. `let particle;`. In this case, all code is visible on one screen, so it matters less, but once the code were to grow beyond that, longer variable names carry more information about their purpose. Related https://google.github.io/styleguide/jsguide.html#naming